src/SuperstateToken.sol

```diff
@@ -67,15 +75,41 @@ abstract contract SuperstateToken is
      */
     uint256[100] private __additionalFieldsGap;
 
+    // TODO: does this shrink the gap / where do I put this stuff?
```

Comment:
i think you shrink the gap by 3. we should test to confirm cc @cridmann

Reply:
also need to consider consts and immutables

Reply:
consts and immutables do not use storage, they are written into the deploy bytecode

Reply:
Yes, thats what I'm saying